Add a test for readable and writable properties

ricea opened the pull request:
Verify that the "readable" and "writable" attributes are correctly available and
configured on TextEncoder and TextDecoder objects.

ricea committed:
Add a test for readable and writable properties

Verify that the "readable" and "writable" attributes are correctly available and
configured on TextEncoder and TextDecoder objects.

ricea commented:
@tyoshino PTAL.

tyoshino commented on tests/stream-properties.html:
```diff
@@ -0,0 +1,33 @@
+
+test(() => {
+  const te = new TextEncoder();
+  assert_idl_attribute(te, 'readable', 'readable property must be present');
```
sorry. it's first time for me to see this assert tool. could you point me to some doc to understand how this works?

http://web-platform-tests.org/writing-tests/testharness-api.html and https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/resources/testharness.js?type=cs&q=assert_idl_attribute&sq=package:chromium&l=1227 don't tell me much.

ricea replied:
Based on the source, it is just an alias for assert_inherits(). I was hoping it would check that the property is enumerable, since the new implementation fixes that, but it doesn't. I could add a check for enumerability in a follow-up change.

tyoshino replied:
Got it. Sounds good.

ricea committed:
Switch from assert_idl_attribute to assert_inherits

assert_idl_attribute() is a deprecated alias for assert_inherits(). Use
assert_inherits() instead.

tyoshino commented:
495ace8bd9c742a2143c12807693c6ebdc43cb33 still lgtm
